backbone/src/main/res/layout/rsb_image_choice_step_layout.xml

```diff
@@ -48,14 +48,30 @@
         app:layout_constraintTop_toBottomOf="@id/imageChoiceStepImageBarrier"
         tools:text="Yes" />
 
-    <ImageView
-        android:id="@+id/imageChoiceImageStepQuestionAnswer"
-        style="@style/rsb_image_choice_step_style"
+    <LinearLayout
```

Comment:
I'd prefer a constraintLayout solution here, I don't think a new viewGroup should be needed.

Reply:
Removed LinearLayout, using the parent ConstraintLayout instead.